src/policy/CMSRucioPolicy/permission.py

```diff
@@ -214,9 +214,9 @@ def _get_rule_size(rules):
 
     try:
         global_usage_per_account = float(config_get(
-            'rules', 'global_usage_per_account', raise_exception=True, default=1e15))
+            'rules', 'global_usage_per_account', raise_exception=True, default=500e12))
```

Comment:
This can be done with the rucio client / cli with the `config` subcommand in `rucio-admin` cli.

You can see this config by
`rucio-admin config get --section rules`

Reply:
That's perfect, thanks a lot @dynamic-entropy 
We had a discussion last Thursday on the coordination meeting and decided to lower it to half.
I've changed the config on production with the cli so I'll close this one